addons/mass_mailing/wizard/mailing_mailing_test.py

```diff
@@ -61,6 +61,7 @@ def send_mail_test(self):
                 'attachment_ids': [(4, attachment.id) for attachment in mailing.attachment_ids],
                 'auto_delete': False,  # they are manually deleted after notifying the document
                 'mail_server_id': mailing.mail_server_id.id,
+                'res_id': self.env.user.id,
```

Comment:
If you give a res_id, it should come with the related model so that there is a linked record, isn't it ?

Reply:
@thjo-odoo I agree with tde, I think we should set the model to have consistent data in db (even if it's not used)